test/NuGet.Core.Tests/NuGet.Commands.Test/MSBuildRestoreUtilityTests.cs

```diff
@@ -3826,6 +3934,239 @@ public void MSBuildRestoreUtility_GetPackageSpec_CPVM_NoVersionChecks()
             }
         }
 
+        /// <summary>
+        /// Verifies taht <see cref="MSBuildRestoreUtility.GetDependencySpec(IEnumerable{IMSBuildItem})" /> applies version overrides correctly depending on whether or not central package management is enabled.
+        /// </summary>
+        /// <param name="isCentralPackageManagementEnabled"><c>true</c> if central pacakge management is enabled, otherwise <c>false</c>.</param>
```

Comment:
```suggestion
        /// <param name="isCentralPackageManagementEnabled"><c>true</c> if central package management is enabled, otherwise <c>false</c>.</param>
```